plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/IRouteChooser.java

```diff
@@ -20,6 +20,17 @@
 public interface IRouteChooser {
 
 	public IRoute chooseRoute(List<IRoute> routes);
+
+	/**
+	 * Returns user's decision to remember the chosen route.
+	 * This method should be called after chooseRoute(List<IRoute).
+	 * 
+	 * @return
+	 */
+	default public boolean isRememberChoice() {
```

Comment:
I think matching style of existing file is more important. There's no real benefit to having public or not. I personally like public modifier, but that's personal preference.